doc/INSTALL-Ubuntu-22.04.md

```diff
@@ -0,0 +1,170 @@
+# Installing Aleph-VM on a server / Ubuntu 22.04 Jammy Jellyfish
+
+## 0. Introduction
+
+For production using official Debian packages.
```

Comment:
A bit weird to mention "Debian" packages when installing on Ubuntu, don't you think? I know that's the name, but maybe "apt" packages could lead to less confusion?

Reply:
https://en.wikipedia.org/wiki/Deb_(file_format)
> Debian packages are standard [Unix](https://en.wikipedia.org/wiki/Unix) [ar archives](https://en.wikipedia.org/wiki/Ar_(Unix)) that (...)